exercises/G-conditionals-2/exercise-1.js

```diff
@@ -6,7 +6,13 @@
   - if number is more or equal to zero, return the word "positive"
 */
 
-function negativeOrPositive(number) {}
+function negativeOrPositive(number) {
```

Comment:
Good stuff! Since this isn't a predicate function , the function is unable to simply return the result of the boolean expression